packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/order/telecom-telephony-alias-portability-order.controller.js

```diff
@@ -248,6 +246,37 @@ angular
         return number;
       };
 
+      self.onRedirectLineSearchChanged = function onRedirectLineSearchChanged(
+        value,
+      ) {
+        if (size(value) < 5) {
+          return;
+        }
+
+        OvhApiTelephony.v6()
+          .searchService({
+            axiom: value,
+          })
+          .$promise.then((d) => {
+            self.autocompleteRedirectLines = d.filter(
+              (service) => service.type === 'line',
```

Comment:
We should ensure that the previous filter `data-available-types="['sip']"` is applied wit this test.